mockdata/authorization-mock/555111001.json

```
@@ -0,0 +1,5 @@
+{
+"Subject": {},
```

Kan jo også få inn litt tabs om du føler for
```suggestion
  "Subject": {},
  "Reportee": {"organizationNumber": "555111001"},
  "Rights": [{"serviceCode": "5755"}]
```